modules/ROOT/pages/create-instance-task.adoc

```diff
@@ -3,53 +3,57 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-In API Manager 2.x, you can use instances of an API in multiple environments. Additionally, you can have multiple instances in the same environment. 
+In API Manager 2.x, instances of an API can exist in multiple environments. Additionally, you can create multiple instances of an API in the same environment. 
 
 You can create one instance of an API that serves as a proxy. You can create another to manage directly as a basic endpoint. You can apply caching policies (for example, to the proxy), and throttling and security policies to the basic endpoint.
 
-After setting up an API version for management, you repeat the process to create another instance. Attempting to set up the same version creates another instance of the version. You can create multiple instances and expand the version information to view details about them.
+After setting up an API version for management, you repeat the process to create another instance. Attempting to configure the same version creates another instance of the version. You can create multiple instances and expand the version information to view details about them.
 
-To create a API instance:
+== Create an API instance
 
-. Select *API Administration > Manage API* to create an API instance from Exchange.
+The following procedure describes creating a single API instance from Exchange, and configuring the API instance as an endpoint with proxy:
+
+. In *API Administration*, click *Manage API*. 
+. Select *Manage API From Exchange*.
 +
-For example, to set up version 1.0 of an API named `jsonplaceholder`:
+To configure version 1.0 of an example API named `jsonplaceholder`:
```

Comment:
Keep in mind that to Create an API Instance an Exchange asset is required.
That means that in Exchange there should be a `jsonplaceholder` asset with 1.0 version.

Maybe this is worth mentioning. (The fact that pre-condition to "Create an API Instance from Exchange", requires the asset)

API Manager supports to manage these asset types: REST API - RAML, REST API - OAS, SOAP API - WSDL, HTTP API.

Reply:
Fixed.